heat/core/linalg/basics.py

```diff
@@ -44,6 +45,118 @@
 ]
 
 
+def cholesky(x: DNDarray, upper: bool = False) -> DNDarray:
+    """
+    Returns the Cholesky decomposition of a Hermitian or symmetric positive-definite matrix.
```

Comment:
Inconsistent documentation (decomposition of a Hermitian doesn't work on complex data types)